templates/flavors/k3s/dual-stack/kustomization.yaml

```diff
@@ -65,18 +80,29 @@ patches:
             namespace: kube-system
           spec:
             targetNamespace: kube-system
-            version: ${CILIUM_VERSION:=1.15.4}
+            version: ${CILIUM_VERSION:=1.16.10}
```

Comment:
Is there a reason for 1.16.10 instead of 1.16.12 for latest patch release on 1.16?

Reply:
Nope, the cilium [addon](https://github.com/linode/cluster-api-provider-linode/blob/main/templates/addons/cilium/cilium.yaml#L12) is at 1.16.10 and hence tried to keep it same.